package.json

```diff
@@ -16,11 +16,12 @@
   ],
   "license": "Apache-2.0",
   "dependencies": {
+    "babel-plugin-async-to-promises": "^1.0.4",
     "babel-plugin-transform-es2015-template-literals": "^6.5.0",
     "babel-plugin-transform-es3-member-expression-literals": "^6.5.0",
     "babel-plugin-transform-es3-property-literals": "^6.5.0",
-    "babel-plugin-transform-jscript": "^6.5.0",
```

Comment:
This was done automatically by `npm install`. Why would the order of the dependencies in the package.json file affect anything?